tests/projects/SelfContained_Trimming_Test/check.ps1

```diff
@@ -19,5 +19,5 @@ $file = Get-Item .\bin\Release\net7.0\win-x64\publish\FSharp.Core.dll
 $file_len = $file.Length
 if (-not ($file_len -eq $expected_len))
 {
-    Write-Error "Test failed with unexpected FSharp.Core length:`nExpected:`n`t${expected_len} Bytes`nActual:`n`t${file_len} Bytes" -ErrorAction Stop
+    Write-Error "Test failed with unexpected FSharp.Core length:`nExpected:`n`t${expected_len} Bytes`nActual:`n`t${file_len} Bytes`nEiter codegen or trimming logic have changed. Please investigate and either update expected size or report an issue." -ErrorAction Stop
```

Comment:
typo: nEiter

Reply:
Also I believe "neither" should go with "nor" (rather than "or")

Reply:
It's not neither. It's a line break `n and a word Either.